src/luarocks/build/rust-mlua.lua

```diff
@@ -84,6 +84,21 @@ function mlua.run(rockspec, no_install)
                 return nil, "Failed installing " .. src .. " in " .. dst .. ": " .. err
             end
         end
+
+        local cwd = dir.path(dir.dir_name(rockspec.local_abs_filename), rockspec.name)
+        local luadir = path.lua_dir(rockspec.name, rockspec.version)
+
+        fs.make_dir(dir.dir_name(luadir))
+        for to, from in pairs(rockspec.build.include) do
```

Comment:
should it be `from, to` ?

Reply:
probably makes more sense! swapped the two keys